dashboard/app/controllers/projects_controller.rb

```diff
@@ -513,4 +515,15 @@ def authorize_load_project!
       authenticate_user!
     end
   end
+
+  def can_remix_project(project_type, channel_id)
+    return true unless RESTRICTED_PUBLISH_PROJECT_TYPES.include?(project_type)
+
+    # Check for restricted share mode. If we are in restricted share mode, it means we cannot remix.
+    source_data = SourceBucket.new.get(channel_id, "main.json")
```

Comment:
nit: do we have a constant for `"main.json"` somewhere we can use?

Reply:
it's defined as a constant in `source_bucket` but unfortunately not exported.